UnitsNet/Scripts/UnitDefinitions/Frequency.json

```diff
@@ -16,7 +16,19 @@
                 }
             ]
         },
-        {
+		{
+            "SingularName": "RadianPerSecond",
```

Comment:
In naval architecture and (sometimes) oceanography rad/s is used for wave frequency.

My guess is that the historic reason is that it makes it easier to go between wave slope (in rad) and frequency.